src/to-sql.ts

```diff
@@ -1183,6 +1183,27 @@ const visitor = astVisitor<IAstFullVisitor>(m => ({
                 m.expr(s.limit.limit);
             }
         }
+
+        if (s.for) {
+            ret.push('FOR ');
```

Comment:
all this could be replaced by `ret.push('FOR ', s.for.toUppercase())` (see comment on ast.ts)